docker/driver-loader/docker-entrypoint.sh

```diff
@@ -120,9 +117,22 @@ while test $# -gt 0; do
     shift
 done
 
+# No opts passed, enable both compile and download
 if [ -z "$has_opts" ]; then
-	ENABLE_COMPILE="true"
-	ENABLE_DOWNLOAD="true"
+    ENABLE_COMPILE="true"
+    ENABLE_DOWNLOAD="true"
+fi
+
+# Default value: auto
+if [ -z "$driver" ]; then
+  driver="auto"
+fi
+
+if [ "$driver" != "auto" ]; then
+  /usr/bin/falcoctl driver config --type $driver
+else
+  # Needed because we need to configure Falco to start with correct driver
+  /usr/bin/falcoctl driver config --type modern_ebpf --type ebpf --type kmod
```

Comment:
Is this the order in which drivers are loaded? I would say we normally prioritize `modern_ebpf`, `kmod`, `ebpf` instead, right?

Reply:
Yes this is the prio order; we decided to go with ebpf before kmod.
We can change it later btw! (i mean, before the release).